src/main/java/net/minecraftforge/common/extensions/IForgeBlock.java

```diff
@@ -920,4 +920,30 @@ default MaterialColor getMapColor(BlockState state, BlockGetter level, BlockPos
     {
         return defaultColor;
     }
+
+    /**
+     * Returns the factor by which jump height is multiplied for any entities standing on this block.
```

Comment:
This documentation doesn't really explain much because there is some additional work going on here. It depends on the jump factor of another block to determine which one is used.

Reply:
That's not true. The logic you mention is in the Entity class.

Reply:
Thanks, didn't really think about the secondary logic the Entity class uses